_layouts/default.html

```diff
@@ -1,68 +1,88 @@
+  <meta name="twitter:description" content="{{ page.description }}" />
+  {% endif %} {% if page.keywords %}
+  <meta name="keywords" content="{{ page.keywords }}" />
+  {% endif %} {% if page.screenshot %}
+  <meta property="og:image" content="{{ page.screenshot | absolute_url }}" />
+  <meta name="twitter:image" content="{{ page.screenshot | absolute_url }}" />
+  <meta name="twitter:image:alt" content="{{ page.title }}" />
+  {% else %}
+  <meta property="og:image" content="{{ '/images/v-logo.jpg' | absolute_url }}" />
+  <meta name="twitter:image" content="{{ '/images/v-logo.jpg' | absolute_url }}" />
+  <meta name="twitter:image:alt" content="{{ page.title }}" />
+  {% endif %}
 
-    <title>{{ page.title }}</title>
-    
-    <link rel="stylesheet" href="/css/sanitize.12.0.1.min.css">
-    <link rel="stylesheet" href="/css/style.css">
+  <title>{{ page.title }}</title>
 
-  </head>
-  <body>
-    <header class="site-header" tabindex="-1">
-      <h1><a href="{{ site.url }}">V_ Innovation</a></h1>
-      <ul>
-        <li><a target="_blank" rel="noreferrer" href="https://www.valtech.com">Valtech.com</a></li>
-        <li><a target="_blank" rel="noreferrer" href="https://www.valtech.com/about/contact-us/">Contact Us</a></li>
-      </ul>
-    </header>
-    <section class="site-content" tabindex="-1">
-      {{ content }}
-    </section>
+  <link rel="stylesheet" href="/css/sanitize.12.0.1.min.css" />
+  <link rel="stylesheet" href="/css/style.css" />  
+</head>
+
+<body onload="checkCookies()">
+  <header class="site-header" tabindex="-1">
+    <h1><a href="{{ site.url }}">V_ Innovation</a></h1>
+    <ul>
+      <li>
+        <a target="_blank" rel="noreferrer" href="https://www.valtech.com">Valtech.com</a>
+      </li>
+      <li>
+        <a target="_blank" rel="noreferrer" href="https://www.valtech.com/about/contact-us/">Contact Us</a>
+      </li>
+    </ul>
+  </header>
+  <section class="site-content" tabindex="-1">{{ content }}</section>
+  <div id="cookiewrapper" class="cookiepopup">
+    <div class="Cookiepopupbox">
+      <div>
+        <p class="adapc">
+          This website may use cookies in order to improve user experience.
+          For more information on how we use cookies, please visit our
+          <a href="https://www.valtech.com/cookie-statement/" title="Cookie Statement" target="_blank"
+            class="privlink">Cookie Statement.</a>
+          By closing this box or continuing to browse the site, you accept our
+          use of such cookies.
+          <a class="btn btn-outline-success acceptbtn" onclick="setCookie('true',2)">Accept cookies</a>
+        </p>
+      </div>
+    </div>
+  </div>
+  {% for script in page.scripts %}
+    {% include {{ script }} %} 
+  {% endfor %}
+
+  {% if jekyll.environment == "development" %}
+  <input type="hidden" value="development" id="site-env"/>
+  {% else %}
+  <input type="hidden" value="production" id="site-env"/>
+  {% endif %}
```

Comment:
Built in feature of Jekyll: https://jekyllrb.com/docs/configuration/environments/